### ISSUE TITLE
Deploy tools on windows can't delete temp folder

### ISSUE THREAD

**Issue by jmartini** [codeforboston/voiceapp311#92]
When running deploy tools on Windows machines, an error is raised due to being unable to remove the created temp directory. 

> Creating temporary build directory ... Traceback (most recent call last):
> File "deploy_tools.py", line 136, in <module>
>    main()
>  File "deploy_tools.py", line 130, in main
>    package_lambda_function()
>  File "deploy_tools.py", line 101, in package_lambda_function
>    os.mkdir(TEMP_DIR_PATH)
>FileExistsError: [WinError 183] Cannot create a file when that file already exists: 'C:\\CodeForBoston\\voiceapp311\\mycity\\mycity\\deploy_tools\\..\\..\\temp'

**Comment by thiteixeira** [comment]
Maybe this will do it. Change [L110](https://github.com/codeforboston/voiceapp311/blob/master/mycity/mycity/deploy_tools/deploy_tools.py#L110) to:

```
try:
    os.mkdir(TEMP_DIR_PATH)
except OSError as e:
    print 'exception thrown'
    shutil.rmtree(TEMP_DIR_PATH, ignore_errors=True)
    os.mkdir(TEMP_DIR_PATH)
```

Sources:
https://stackoverflow.com/questions/303200/how-do-i-remove-delete-a-folder-that-is-not-empty-with-python
and
https://stackoverflow.com/questions/22027508/python-getting-a-windowserror-instead-of-an-ioerror

**Comment by curranjm** [comment]
Working on this I got rid of the code that was checking the pip version as we shouldn't need it and closed issue #93 .
